### PR TITLE
Add ColdTurkey.app v3.1

### PR DESCRIPTION
**Pull request by jinntrance**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/cold-turkey.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'cold-turkey' do
+  version '3.1'
+  sha256 :no_check
+
+  url 'https://getcoldturkey.com/files/Cold_Turkey_Mac_Installer.pkg'
+  name 'Cold Turkey'
+  homepage 'https://getcoldturkey.com/'
+
+  pkg 'Cold_Turkey_Mac_Installer.pkg'
```
Comment (commitay) [inline Casks/cold-turkey.rb]: Add [`uninstall pkgutil`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md#uninstall-key-pkgutil)

Example:
https://github.com/caskroom/homebrew-cask/blob/9c23c72d088ee016720d0409f686f229ad19b90f/Casks/vfuse.rb#L11-L13

File: Casks/cold-turkey.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'cold-turkey' do
+  version '3.1'
+  sha256 :no_check
```
Comment (commitay) [inline Casks/cold-turkey.rb]: Please add the `sha256`

**Review by commitay — CHANGES_REQUESTED** [review]

**Review by commitay — CHANGES_REQUESTED** [review]
Please rename the Cask to `cold-turkey-blocker`

[Simplified Names of pkg-based Installers](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md#simplified-names-of-pkg-based-installers)

**Comment by jinntrance** [comment]
@commitay Thank you for your comments.  I've changed what you requested.

**Review by commitay — APPROVED** [review]

**Comment by commitay** [comment]
Thanks for the PR!

**Pull request merged**